src/content/docs/developer/modules/other/workflow-step-type.mdx

```diff
@@ -0,0 +1,553 @@
+
+Example webhook payload for the `string.status_on_step.recalculation_triggered` event:
+
+```json
+{
+  "events": [
+    {
+      "event": "string.status_on_step.recalculation_triggered",
+      "stringStatus": {
+        "status": "NEED_PROCESS",
+        "output": "",
+        "organizationId": "200007777",
+        "translation": {
+          "id": 1106423,
+          "identifier": "058eb6ea2bdcc79a6a7208783c8bfb50",
+          "key": "string_1",
+          "text": "Not all videos are shown to users. See more",
+          "type": "text",
+          "context": "string_1",
+          "maxLength": "50",
+          "isHidden": false,
+          "isDuplicate": false,
+          "masterStringId": null,
+          "revision": 1,
+          "hasPlurals": false,
+          "labelIds": [],
+          "url": "https://umbrella.crowdin.com/editor/173/743/en-et#1106423",
+          "createdAt": "2024-10-29T10:47:13+00:00",
+          "updatedAt": null,
+          "file": {
+            "id": 743,
+            "name": "umbrella_app.xml",
+            "title": null,
+            "type": "android8",
+            "path": "/umbrella_app.xml",
+            "status": "active",
+            "revision": "1",
+            "branch": {
+              "id": null
+            },
+            "directory": {
+              "id": null
+            },
+            "project": null
+          },
+          "project": {
+            "id": 173,
+            "userId": 1,
+            "sourceLanguageId": "en",
+            "targetLanguageIds": [
+              "uk",
+              "et"
+            ],
+            "identifier": "d3026ae4cff9820bc140a210d23b35ad",
+            "name": "Project Name",
+            "createdAt": "2024-10-25T14:37:47+00:00",
+            "updatedAt": "2024-10-25T14:37:47+00:00",
+            "lastActivity": "2025-01-30T09:32:58+00:00",
+            "description": "",
+            "url": "https://umbrella.crowdin.com/u/projects/173",
+            "cname": null,
+            "languageAccessPolicy": null,
+            "visibility": null,
+            "publicDownloads": null,
+            "logo": "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAJYAAACWCAYAAAA8AXHiAAAACXBIWXMAAA7EAAAOxAGVKw4bAAACJUlEQVR4nO3bL06cQRyA4VnSaxAMd0BgUHuAZs+BJVnHAWpJFYrgMKieoyj+VDQgmjZps2E9PUEd736b7fMc4PeNeDMz4pvZ4nT5NuCd7U29AHaTsEgIi4SwSAiLhLBICIuEsEgIi4SwSAiLhLBICIuEsEgIi4SwSAiLhLBICIuEsEgIi4SwSAiLhLBICIuEsEgIi4SwSAiLhLBICIuEsEgIi4SwSAiLhLBICIuEsEgIi4SwSAiLhLBICIuEsEgIi4SwSAiLxIepF7BJH+cnY358tPHvfr1/GhfXNxv/7pTsWCSERUJYJIRF4r+6vP/L6nU97h6+ZfNffvzMZm8rYY0xfv3+M65uv0y9jJ3iKCQhLBLCIiEsEsIiISwSwiIhLBLCIiEsEsIiISwSwiIhLBLCIiEsEsIiISwSfk0eYxwe7I/P52fJ7E+X1+Px+3Mye5vZsUgIi4SwSAiLhMv7aB+srtbrZO62E9bwYLXgKCQhLBLCIiEsEsIiISwSwiIhLBLCIiEsEsIiISwSwiIhLBLCIiEsEsIiISwSwiIhLBLCIiEsErPF6fJt6kWwe+xYJIRFQlgkhEVCWCSERUJYJIRFQlgkhEVCWCSERUJYJIRFQlgkhEVCWCSERUJYJIRFQlgkhEVCWCSERUJYJIRFQlgkhEVCWCSERUJYJIRFQlgkhEVCWCSERUJYJIRFQlgkhEVCWCSERUJYJIRFQlgkhEVCWCT+AlFELAxLqwmKAAAAAElFTkSuQmCC",
```

Comment:
Let's trim this long value, there is no need in full example of base64 image here